src/components/layout/Layout.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Outlet } from "react-router-dom";
+
+import "./Layout.css";
+
+import NavigationBar from "../navigation/NavigationBar";
+import TopNavigation from "../top-navigation/TopNavigation";
```

Comment:
If you name `TopNaviagation` file to `index.jsx` then the import statement will be changed from 
`../top-navigation/TopNavigation` to this `../top-navigation` as by default index file gets imported. This way you can avoid repetation.